src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -0,0 +1,67 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=method-hidden
+#pylint: disable=line-too-long
+#pylint: disable=bad-continuation
+from __future__ import print_function
+
+import os
+import time
+
+from azure.cli.core._util import CLIError
+from azure.cli.core.test_utils.vcr_test_base import (ResourceGroupVCRTestBase, JMESPathCheck,
+                                                     NoneCheck, VCRTestBase)
+
+class SqlServerMgmtScenarioTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(SqlServerMgmtScenarioTest, self).__init__(__file__, test_method, resource_group='cli-test-sql-mgmt')
+        self.sql_server_names = ['cli-sql-12345-a', 'cli-sql-12345-b']
```

Comment:
Make the server name more readable. For example, 'cliautomation01'. It helps us to identify server in the test subscription in case they're not cleaned up.